proto/redpanda/api/connect/v1alpha1/status.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package redpanda.api.connect.v1alpha1;
+
+option go_package = "internal/protoconnect";
+
+// ConnectionError describes a specific connection failure.
+message ConnectionError {
+  string message = 1; // The error message.
+  repeated string path = 2; // The path of the connector in the config.
+  optional string label = 3; // An optional label given to the connector.
+}
+
+// ExitError describes an error encountered that caused the instance to exit.
+message ExitError {
+  string message = 1; // The error message.
+}
+
+// StatusEvent describes the current state of an individual connect instance,
+// which is self-reported periodically.
+message StatusEvent {
+  enum Type {
+    // The status has not been specified.
+    TYPE_UNSPECIFIED = 0;
+    // An instance has parsed a config and is now attempting to run a pipeline.
+    TYPE_INITIALISING = 1;
```

Comment:
Let's use american english spelling. It's the standard for our docs and the rest of our code.

https://github.com/redpanda-data/docs-site/blob/main/meta-docs/STYLE-GUIDE.adoc

Reply:
Fixed, my bad